io_scene_dae/export_dae.py

```diff
@@ -61,27 +61,26 @@ def snap_tup(tup):
 
 
 def strmtx(mtx):
-    s = " "
+    s = ""
     for x in range(4):
         for y in range(4):
-            s += str(mtx[x][y])
-            s += " "
-    s += " "
+            s += " {}".format(mtx[x][y])
+    s = " {} ".format(s)
     return s
```

Comment:
Well spotted. Fixed it with a new commit.
Or should I add this change to the previous commit?